src/services/dataService/__tests__/listItem.test.js

```diff
@@ -1,3 +1,42 @@
 const test = require('../../../test')
 const seedData = require('../../../test/seedData')
-const {getUser, getList, findLists} = require('../../dataService')
+const {findListItems, createListItem, deleteListItem, updateListItem, getListItem} = require('../../dataService/listItemFunctions')
+
+test('dataService', function(t) {
+  return seedData().then(() => {
+    t.test('Get item by id', function (st) {
+      getListItem(1).then(function (listsItem) {
+        st.equal(listsItem.id, 1, 'Returned list item id' )
+        st.end()
+      })
+    })
+    t.test('Create new list item', function (st) {
+      createListItem({listId: 2, description: 'blah'}).then(function (listsItem) {
+        st.equal(listsItem.description, 'blah', 'Create list item and returned description' )
+        st.equal(listsItem.listId, 2, 'New list item belongs to sepecified list')
+        st.end()
+      })
+    })
+    t.test('Delete list item', function (st) {
+      deleteListItem(1).then(function () {
+        getListItem(1).then(function (listItem) {
+          st.equal(listItem, null, 'Deleted a lists item' )
+          st.end()
+        })
+      })
+    })
+    t.test('Update list item', function (st) {
+      updateListItem(2,{description: 'blarg'}).then(function (listItem) {
+        st.equal(listItem.description, 'blarg', 'Updated list item description and returned new description' )
```

Comment:
Would be helpful to first get the list item to capture the original value, then assert that it changed.